lib/net/ssh/transport/openssl.rb

```diff
@@ -245,6 +245,10 @@ def ssh_type
           "ecdsa-sha2-#{CurveNameAliasInv[group.curve_name]}"
         end
 
+        def ssh_signature_type
```

Comment:
```suggestion
alias ssh_signature_type ssh_type
````
It might be more clear to just use ruby's `alias` keyword for this.

Reply:
Changed the code according to your suggestion. Also changed other `ssh_signature_type` methods as well.